build.gradle

```diff
@@ -0,0 +1,36 @@
+// plugins: add functionality to your build
+plugins {
+    id 'java'   // adds capabilities like compiling java code and running tests
+    id 'application'
+    id 'org.openjfx.javafxplugin' version '0.0.10'
+}
+
+// build metadata: information about the build
+group 'com.skwok.pacman'
+version '1.0-SNAPSHOT'
+
+// repositories: where your build's dependencies are downloaded from
+repositories {
+    mavenCentral()
+}
+
+// dependencies: artifacts that are required in order to build the project
+dependencies {
+    testCompile group: 'junit', name: 'junit', version: '4.12'
```

Comment:
The `testCompile` configuration is deprecated and `testImplementation` should be used instead (the deprecated configurations have been removed in Gradle 7).